app/views/events/_monthview.html.erb

```diff
@@ -52,6 +52,9 @@
               elsif date[:events].size > 4
                 day_class = "more_events"
               end
+              if date[:date].to_date == Date.current
+                day_class += " current_day"
```

Comment:
This will error if `day_class` is still `nil`. Since there is no `else` clause above it setting it to a string, we can't guarantee it isn't `nil`. You could change the `nil` to an empty string (and the others to concatenate) or do something like `day_class = (day_class || "") + " current_day"`.

Even better, there is a [class names helper](https://www.bigbinary.com/blog/rails-6-1-introduces-class_names-helper) so we could just concatenate a list instead; that's the most Rails way to do it. This also avoids an extra preceding space in the class list on days without events.